index.js

```diff
@@ -50,6 +50,6 @@ module.exports = function (cb) {
 			cb(null, false);
 		}, timeout);
 
-		udpSocket.unref();
+		udpSocket.close();
```

Comment:
You can't immediately close here after sending. I think you should move that close call into the `setTimeout` above.